test/cross-browser/reference.html

```diff
@@ -22,36 +20,51 @@
 
     <!-- traces -->
     <script>
-        var childWin;
+        var clientLogger;
+
+        function createLocalStorageLogger() {
+            return {
+                debug: function() {
+                    localStorage.setItem(
```

Comment:
I'm leaning more towards the `setItem`/`getItem` implementation.  I read through the SO article you mentioned, I feel like localStorage is not a traditional array object (as mentioned by one answer, length is not settable).  So I feel that it's wise not to treat it like one... Yeah?